src/topology/tactic.lean

```diff
@@ -0,0 +1,77 @@
+/-
```

Comment:
Two related questions: 
- should we put this in `src/tactic/continuity.lean`?
- Also, should we add this to `tactic/default.lean`? 

@robertylewis Do you have thoughts on how we should organize "theory-specific" tactics like this? I think there might be some tactics living in category_theory already.

Reply:
I wouldn't add this to `tactic/default.lean` but just import it in `topology/basic`. You will never need it if you don't have `topology.basic` imported.

Reply:
> @robertylewis Do you have thoughts on how we should organize "theory-specific" tactics like this? I think there might be some tactics living in category_theory already.

With the tactic doc entries, I don't think there's a big problem with having them appear scattered throughout the library. There's a central place to find all tactics and they link to their location in source. A special purpose tactic that won't be imported by other tactics could certainly live in the appropriate theory folder.